Securitytxt/Model/Config/Backend/Validate.php

```diff
@@ -0,0 +1,137 @@
+            $hiring = $otherInfoFields['hiring']['value'];
+            $policy = $otherInfoFields['policy']['value'];
+
+            if ($isEnabled) {
+                if ($contactEmail == '' && $contactPhone == '' && $contactWebpage == '') {
+                    throw new ValidatorException(__('At least one contact information is required.'));
+                }
+            }
+
+            if ($contactEmail != '') {
+                if (!filter_var($contactEmail, FILTER_VALIDATE_EMAIL)) {
+                    throw new ValidatorException(
+                        __('Contact Information: Email validation failed. Please enter in correct format.')
+                    );
+                }
+            }
+
+            if ($contactPhone != '') {
+                if (!$this->validatePhone($contactPhone)) {
+                    throw new ValidatorException(
+                        __('Contact Information: Phone number validation failed. Please enter in correct format.')
+                    );
+                }
+            }
+
+            if ($contactWebpage != '') {
+                if (!$this->validateUrl($contactWebpage)) {
+                    throw new ValidatorException(
+                        __('Contact Information: Contact Page URL should be in correct format and must start with HTTPS.')
+                    );
+                }
+            }
+
+            if ($encryption != '') {
+                if (!$this->validateUrl($encryption)) {
+                    throw new ValidatorException(
+                        __('Other Information: Encryption URL should be in correct format and must start with HTTPS.')
+                    );
+                }
+            }
+
+            if ($acknowledgements != '') {
+                if (!$this->validateUrl($acknowledgements)) {
+                    throw new ValidatorException(
+                        __('Other Information: Acknowledgements URL should be in correct format and must start with HTTPS.')
+                    );
+                }
+            }
+
+            if ($hiring != '') {
+                if (!$this->validateUrl($hiring)) {
+                    throw new ValidatorException(
+                        __('Other Information: Hiring URL should be in correct format and must start with HTTPS.')
+                    );
+                }
+            }
+
+            if ($policy != '') {
+                if (!$this->validateUrl($policy)) {
+                    throw new ValidatorException(
+                        __('Other Information: Policy URL should be in correct format and must start with HTTPS.')
+                    );
+                }
+            }
+
+        }
+        parent::validateBeforeSave();
+        return $this;
+    }
+
+    /**
+     * @param $phone
+     * @return bool
+     */
+    protected function validatePhone(string $phone)
+    {
+        $filtered_phone_number = filter_var($phone, FILTER_SANITIZE_NUMBER_INT);
```

Comment:
Generally we do not use snake case names in Magento code.

Reply:
Filtering with the INT may not be applicable in some cases. Also, phone number may be formatted in different ways for different regions